Makefile.common

```diff
@@ -109,7 +99,7 @@ endif
 %: common-% ;
 
 .PHONY: common-all
-common-all: precheck style check_license lint unused build test
+common-all: precheck style check_license lint build test unused
```

Comment:
@nevill I  think we should not make changes to the `Makefile.common` file, overriding them in the `Makefile` should do because the `Makefile.common` is updated by @prombot 

@krasi-georgiev can comment more on this if we decide to change this.

Reply:
Yes, I also noticed PR #196 , which is where `Makefile.common` came in the project. I will try to make my changes into `Makefile` instead.

IMO `Makefile.common` is quite redundant, for example, `GO111MODULE` is always `on` in this project. It's not friendly to new contributors as well.

More interestingly, from [promu document](https://docs.google.com/document/d/1Ql-f_aThl-2eB5v3QdKV_zgBdetLLbdxxChpy-TnWSE/edit#heading=h.24x0bg1hyuak)
> The Makefile.COMMON approach was abandoned for Prometheus and Alertmanager eventually.